### PR TITLE
✨Add disable-double-tap attribute for amp-pan-zoom

### PR DESCRIPTION
**Pull request by cathyxz**
Partially addresses https://github.com/ampproject/amphtml/issues/17416. This adds an attribute `disable-double-tap` to allow users of this component to disable double tap zoom to remove the 200ms latency necessary to preserve doubletap detection. This does not disable double-tap zoom by default. Open to feedback on naming.

### CODE REVIEW

**Comment by Gregable** [comment]
validator changes lgtm

File: extensions/amp-pan-zoom/validator-amp-pan-zoom.protoascii [hunk 1]
```diff
@@ -29,6 +29,10 @@ tags: {  # <amp-pan-zoom>
   html_format: EXPERIMENTAL
   tag_name: "AMP-PAN-ZOOM"
   requires_extension: "amp-pan-zoom"
+  attrs: {
+    name: "disable-double-tap"
```
Comment (honeybadgerdontcare) [inline extensions/amp-pan-zoom/validator-amp-pan-zoom.protoascii]: please alphabetize by attrs.name (e.g. after `controls`)

Comment (cathyxz) [reply extensions/amp-pan-zoom/validator-amp-pan-zoom.protoascii]: Oh oops. Idk why I misread the alphabet here.

**Comment by spacedino** [comment]
Double check to make sure this plays well with browser tap to zoom features.

**Comment by kul3r4** [comment]
qq, what does "controls" attribute means?

**Comment by cathyxz** [comment]
It doesn't mean anything anymore. It was removed because having controls is now the default behavior. @kul3r4

**Pull request merged**